src/Microsoft.AspNet.Mvc.ViewFeatures/ViewComponentResult.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNet.Mvc.ViewFeatures;
+using Microsoft.AspNet.Mvc.ViewFeatures.Internal;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.OptionsModel;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNet.Mvc
+{
+    public class ViewComponentResult : ActionResult
```

Comment:
Needs docs.

This has all the same options as `ViewResult` except:
- `ViewName` -> `ViewComponentName` **or** `ViewComponentType`
- `Model` -> `Arguments`